examples/lib/stories/effects/move_effect_example.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:math';
+
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/src/effects2/move_effect.dart'; // ignore: implementation_imports
+import 'package:flame/src/effects2/standard_effect_controller.dart'; // ignore: implementation_imports
```

Comment:
We'll have to remember to remove these ignores later when we export everything from the new effects system.